ui-admin/src/study/surveys/SurveyEnvironmentDetailModal.tsx

```diff
@@ -37,6 +40,61 @@ export default function SurveyEnvironmentDetailModal(props: SurveyEnvironmentDet
 
   const mostRecentVersion = configuredSurveys[configuredSurveys.length - 1]?.survey?.version
 
+  const columns = useMemo((): ColumnDef<StudyEnvironmentSurvey>[] => {
+    return [{
+      id: 'versionName',
+      header: '',
+      cell: ({ row }) => row.original.survey.version
+    }, {
+      id: 'launchDate',
+      header: 'Launch date',
+      cell: ({ row }) => instantToDateString(row.original.createdAt)
+    }, {
+      id: 'numParticipants',
+      header: '# participants currently assigned',
+      cell: ({ row }) => participantTasks
+        .filter(task => task.targetAssignedVersion === row.original.survey.version).length
+    }, {
+      id: 'actions',
+      header: '',
+      cell: ({ row }) => <>
+        { (hasParticipantsOnOldVersions && row.original.survey.version === mostRecentVersion) &&
+          <span className="ms-4">
+            <Button variant="secondary" outline={true} onClick={updateAllTasks}>
+              Update all to version {mostRecentVersion}
+            </Button>
+            <InfoPopup content={`Update all participants in the ${studyEnvParams.envName} 
+            environment to have the latest version of this form.  
+            This means when they revisit, they will see the latest version, with any prior
+            answers already filled-in`}/>
+          </span>
+        }
+      </>
+    }]
+  }, [stableId, participantTasks.length])
+
+  const table = useReactTable({
+    data: configuredSurveys,
+    columns,
+    enableRowSelection: true,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    debugTable: true
```

Comment:
```suggestion
    getSortedRowModel: getSortedRowModel()
```

nit: remove debug statements